cellmerged.go

```diff
@@ -33,6 +33,24 @@ func (f *File) GetMergeCells(sheet string) ([]MergeCell, error) {
 	return mergeCells, err
 }
 
+// RemoveMergeCell removes a MergeCell.
+func (f *File) RemoveMergeCell(sheet string, m MergeCell) error {
```

Comment:
I propose to use `UnmergeCells` as the function name.

Reply:
I think you mean `UnmergeCell`? It unmerges a single mergecell.